app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx

```diff
@@ -173,10 +173,7 @@ const SendBitcoinDestinationScreen = ({
     skip: !useIsAuthed(),
   })
 
-  useRealtimePriceQuery({
-    fetchPolicy: "network-only",
-    skip: !useIsAuthed(),
-  })
+  useRealtimePriceWrapper()
```

Comment:
Can you check to see if this makes a network request with each letter you type?

Reply:
sure let me check

Reply:
nope not seen such issue

Reply:
Sounds good